shared/utility/motion/splines/Footstep.cpp

```diff
@@ -85,15 +85,15 @@ namespace utility::motion::splines {
         stepFromSupport(tmpDiff);
     }
 
-    Eigen::Vector3f Footstep::poseAdd(const Eigen::Vector3f& pose, const Eigen::Vector3f& diff) const {
+    Eigen::Vector3f Footstep::poseAdd(const Eigen::Vector3f& pose, const Eigen::Vector3f& diff) {
```

Comment:
This seems strange? The function is not modifying any of the class state, so it should remain const?

Reply:
These were made `static` by this PR, so they couldn't be `const`. In the follow up PR #439 at the advice of Trent, I made them free functions in that namespace